homeworks/andrey.makarenko_Baksckua/3-functions/functions1.js

```diff
@@ -0,0 +1,5 @@
+function getIfBasis(basis, exponent) {
```

Comment:
This code doesn't work. Please, review the code before pushing it.
Also, you need to fix it